docs/overview/script-storage.md

```diff
@@ -1,33 +1,52 @@
+> [!IMPORTANT]
+> SharePoint support for Office Scripts is being rolled out and is not available to everyone. It's slowly released to larger numbers of users to ensure that it's working as expected. This feature is subject to change based on your feedback.
 
-Unless you share your scripts, no one else can access them. Your OneDrive settings control the shared access and permissions for all script **.osts** files, independent of any Excel settings. Scripts can't be linked from a local disk or custom cloud locations. Office Scripts only recognizes and runs a script if it's in your OneDrive folder or shared with the workbook.
+Office Scripts are stored as **.osts** files in your Microsoft OneDrive or a team SharePoint folder. They are stored separately from a workbook. To give others access, [share the script with an Excel workbook](excel.md#share-office-scripts). This means you're linking the script with the file, not attaching it. Whoever has access to the Excel file will also be able to view, run, or make a copy of the script.
 
-## File storage
+Excel only recognizes and runs a script if it's in your OneDrive folder, a team Sharepoint folder, or shared with the workbook.
 
-You Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
+## OneDrive
+
+The default behavior is that Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
 
 Scripts that are shared with one of your workbooks remain in the script creator's OneDrive. They are not copied to any of your local or OneDrive folders when you run the shared script in Excel. The **Make a Copy** button of the Code Editor saves a separate copy of the script in your OneDrive. Changes to the copy don't affect the original script.
 
-### Restore deleted scripts
+Unless you share your personal scripts, no one else can access them. Your OneDrive settings control the shared access and permissions for all script **.osts** files, independent of any Excel settings. Scripts can't be linked from a local disk or custom cloud locations.
+
+## SharePoint
+
+Office Scripts that are saved to a SharePoint site become team scripts. You and members of your organization with the appropriate access can run and edit scripts from SharePoint. You'll also see team scripts appear in the **Automate** tab's Script Gallery.
```

Comment:
How should we reconcile teams from Microsoft Teams here?

Reply:
I'll remove the term "team" here as much as possible.